Changes.md

```diff
@@ -8,6 +8,7 @@ Improvements
 - CropWindowTool : Added <kbd>`Alt` + <kbd>`C` for toggling both the crop window tool and the relevant crop window `enabled` plug.
 - TaskList, FrameMask : Reimplemented in C++ for improved performance.
 - Cache : Increased default computation cache size to 8Gb. Call `Gaffer.ValuePlug.setCacheMemoryLimit()` from a startup file to override this.
+- Dispatcher : Improved internal `dispatch()` performance, with once benchmark showing around a 3x speedup.
```

Comment:
I wonder whether it would be worth being a bit more explicit - it's up to 10x in tests of pure overhead, but probably much lower than 3x even in any realistic production test, where slow dispatches are usually the result of other problems, not this overhead. I'm just thinking of it because I've already gotten excited queries from IE people who heard about this and were asking whether it could help with their painfully slow 3000 frame dispatches, and told them ( I think correctly? ) that if their 3000 frame dispatches are painfully slow, something else is going wrong, and this probably won't help too much.

Reply:
Not that this is really worth worrying about, but I had an interesting idea, so I might as well record it: this might be an unusual situation where it would be more useful to summarize with an absolute performance number, rather than a ratio. The ratio doesn't really generalize anyway, since it totally depends on how many frames you are dispatching at once. What if we described this as "saves x seconds on a dispatch of 100000 frames"? I don't actually know the number, but since testBasicBatcherPerformance now runs in "roughly the ballpark we want", and improved by 10X, I would assume that's something like 20 seconds to 2 seconds. Since it doesn't scale linearly, 100000 would be something less than 10 seconds, so something like "saves 7 seconds on a dispatch of 100000 frames"? That would clearly communicate how much you can expect on a 100000 frame dispatch that takes 10 seconds ( and how little you can expect on a 3000 frame dispatch that takes 2 minutes ).